.travis.yml

```diff
@@ -53,6 +53,29 @@ stages:
 jobs:
   include:
     ########## Test Stage ##########
+    
+    - &default
+      stage: test
+      env: DESC="Python 3.6 tests" HV_REQUIREMENTS="unit_tests" PYTHON_VERSION=3.6
```

Comment:
Are we using these environment variables anywhere for this job? if not using, then remove them